### DISCUSSION TITLE
add gene expression in plotbrowser track

### DISCUSSION THREAD

**Discussion by yamihn** [GreenleafLab/ArchR#1619] [Questions / Documentation]
Hello everyone. I apologize for my question, I was wondering if there was a way to include gene expression in that specific region in the coverage plot. I place an image here as an example from signac. I know there is a function "usematrix", but I would to show the expression as a dot or a violin, for example, near each coverage. 
thank you 

![unnamed-chunk-11-1](https://user-images.githubusercontent.com/109409803/189547130-9f28aaac-8aa2-4d38-a95d-4be73b8ffb36.png)

**Comment by rcorces** [comment]
this is not yet available but is on the list to be added.
